src/Symfony/Component/Routing/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -105,8 +105,16 @@ public function match(\$pathinfo)
     {
         \$allow = array();
         \$pathinfo = rawurldecode(\$pathinfo);
+        \$trimmedPathinfo = rtrim(\$pathinfo, '/');
         \$context = \$this->context;
         \$request = \$this->request;
+        \$requestMethod = \$isLikeGetMethod = \$context->getMethod();
+        \$schema = \$context->getScheme();
+
+        if ('HEAD' === \$requestMethod) {
+            \$isLikeGetMethod = 'GET';
```

Comment:
The `is` prefix in this variable name looks wrong. Maybe it's too late, but what if we rename `$isLikeGetMethod = 'GET'` to `$isSafeMethod = true`

We had some discussions about what a "safe method" is in HTTP, but the section 9.1.1 of RFC 2616 mentions `GET` and `HEAD` as safe methods: http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html